src/server/scripts/Commands/cs_misc.cpp

```diff
@@ -645,7 +645,13 @@ class misc_commandscript : public CommandScript
 
     static bool HandleDismountCommand(ChatHandler* handler, char const* /*args*/)
     {
-        Player* player = handler->GetSession()->GetPlayer();
+        Player* player = handler->getSelectedPlayerOrSelf();
+        if (!player)
```

Comment:
Logic dictates that nobody should run player commands through the console, but you're not wrong I guess.

Good stuff. 🎉